src/config-util/getConfig.js

```diff
@@ -19,7 +19,8 @@ function isSimple(config) {
     - {}
     - [
         {
-          "filters": ["src/*.js", "!src/*.ignore.js"],
+          "includes": ["src/*.js"],
+          "excludes": ["!src/*.ignore.js"],
```

Comment:
A super early review.. ✌️ 😅 

Won't this possibly cause confusion (negating the negated things)?
IMO omitting `!` to be just `"excludes": ["src/*.ignore.js"]` is better.

What do you think?

Reply:
Ya.. my bad. That was by mistake.